### PR TITLE
feat(social-media/publish-post): Expose original error

### PR DESCRIPTION
**Pull request by jnv**
## Description

This adds `original` field to social-media/publish-post and prepares new field `code` as a normalized error code across providers (code isn't yet implemented but can be continuously added to the maps).

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [x] I haven't repeated the code. (DRY)
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: grid/social-media/publish-post/maps/publish-media-post.ts [hunk 1]
```diff
@@ -36,7 +36,7 @@ export const publishMediaPostTest = (
             input: {},
           });
 
-          expect(result.isOk()).toBeTruthy();
+          // expect(result.isOk()).toBeTruthy();
```
Comment (janhalama) [inline grid/social-media/publish-post/maps/publish-media-post.ts:39]: I suggest to remove the expect completely.

File: grid/social-media/publish-post/profile.supr [hunk 2]
```diff
@@ -107,3 +120,47 @@ model MediaItem {
...
+  The request is understood, but it has been refused or access is not allowed.
+  """
+  Unauthorized
+
+  Timeout
+  ServerError
+  Spam
+  RestrictedProfile
+  PayloadTooLarge
+  UnsupportedFormat
```
Comment (janhalama) [inline grid/social-media/publish-post/profile.supr:159]: I wonder if we need both `UnsupportedFormat` and `BadRequest` error codes. Is not this duplicity.

**Review by janhalama — APPROVED** [review]
Looks good 👍 . I have few minor comments.

File: grid/social-media/publish-post/profile.supr (hunk 2 above)
Comment (jnv) [reply grid/social-media/publish-post/profile.supr:159]: Good point, this error relates to the media format (e.g. I am uploading a picture or video with some unsupported encoding).

I will rename this to `UnsupportedAttachmentFormat`

**Pull request merged**